extension/src/popup/components/manageAssets/SearchAsset/index.tsx

```diff
@@ -71,15 +72,14 @@ const ResultsHeader = () => {
 
 export const SearchAsset = ({ setErrorAsset }: SearchAssetProps) => {
   const { t } = useTranslation();
-  const { network } = useSelector(settingsNetworkDetailsSelector);
+  const networkDetails = useSelector(settingsNetworkDetailsSelector);
   const [assetRows, setAssetRows] = useState([] as ManageAssetCurrency[]);
   const [maxHeight, setMaxHeight] = useState(0);
   const [isSearching, setIsSearching] = useState(false);
   const [hasNoResults, setHasNoResults] = useState(false);
   const ResultsRef = useRef<HTMLDivElement>(null);
 
-  // TODO IN THIS PR
-  const isTestnet = network === "TESTNET";
+  const isNetworkTestnet = isTestnet(networkDetails);
```

Comment:
similarly `isTestnet` and `isMainnet` needs `networkDetails` to determine a user's network